app/Console/Commands/Sync.php

```diff
@@ -61,7 +61,7 @@ public function __construct()
 	 *
 	 * @throws ExternalLycheeException
 	 */
-	public function handle(Exec $exec): int
+	public function handle(): int
```

Comment:
Please also remove the PHPdoc comment `@param ...` for the removed parameter.

Reply:
woops